ddp-workspace/projects/ddp-sdk/src/lib/components/user/userMenu.component.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { UserMenuComponent } from './userMenu.component';
+import { Auth0AdapterService, mockComponent, SessionMementoService } from 'ddp-sdk';
```

Comment:
Suggest replacing the import path with absolute one.

Reply:
but what's the benefit?

Reply:
I guess it can potentially lead to a circular dependency.
Not sure if it is critical for tests.

Reply:
I'm not sure it's possible to have a circular dependency issue since sdk public-api.ts file doesn't have any imports of unit tests. We have such troubles with components since sdk public-api exports components.